colinMarkDown.Rmd

```diff
@@ -0,0 +1,105 @@
+---
+title: "Tropical Storm Colin Code"
+output: html_document
+---
+
+## Data retrieval function
+Not sure if we want to include all this or not, but hooray it *works*!
+
+
+```{r global_options, warning=FALSE, message=FALSE}
+library(dataRetrieval)
```

Comment:
actually, we should have those in there because this is a blog post and they will need to see what those functions are